tests/Feature/ExampleTest.php

```diff
@@ -12,10 +12,5 @@ class ExampleTest extends TestCase
      *
      * @return void
      */
-    public function testBasicTest()
-    {
-        $response = $this->get('/');
 
```

Comment:
sao lai xoa di the nay em

Reply:
nãy em bị CI nó kêu lỗi chỗ đấy khi comment trước chị Huyền bảo em xóa cái route welcome mặc định của laravel đi thôi anh và chị bảo em xóa cái đó đi